src/jank/clojure/core.jank

```diff
@@ -431,6 +431,92 @@
                  #{ o }#
                );"))
 
+(defn conj!
+  ([]
+   (transient []))
+  ([coll]
+   coll)
+  ([coll x]
+   (native/raw "__value = visit_object
+                (
+                  [=](auto const typed_t, auto const head) -> object_ptr
+                  {
+                    using T = typename decltype(typed_t)::value_type;
+
+                    if constexpr(behavior::persistentable<T>)
+                    { return typed_t->cons_in_place(head); }
```

Comment:
The behavior for `cons_in_place` is `behavior::consable_in_place`. The behavior for `persistent` is `behavior::persistentable`.